devtools_options.yaml

```diff
@@ -0,0 +1,3 @@
+description: This file stores settings for Dart & Flutter DevTools.
```

Comment:
:question: Does it make sense to include this in VCS? Or should we .gitignore it?